### ISSUE TITLE
[MFTF] User views licensed images in the grid

### ISSUE THREAD

**Issue by filmaj** [magento/adobe-stock-integration#442]
# Description
Implement MFTF test to cover scenario: "User views licensed images in the grid" (story 22, #309)

#  Preconditions

1. Admin User is logged into Admin Panel
2. User signs into Adobe IMS

# Test steps

3. Open Adobe Stock Images Grid 
4. User expands image preview of first image
5. User clicks License and Save button
6. User clicks OK in the confirmation dialog
7. Open Adobe Stock Images Grid
8. Verify that first image has a licensed label

**Comment by drpayyne** [comment]
Waiting on @sivaschenko to get back on how to obtain a new image every time. 

Also, two scenarios are considered here. One for preview saved already, and one for brand new image.

**Comment by filmaj** [comment]
We shouldn't be licensing images during the tests, so with that in mind, I think we can consider this issue in its current form a duplicate of #440.

Perhaps we could modify this test such that:

- assume the Adobe IMS account used for the test has a pre-licensed image
- assume the Media Gallery does not have any saved images
- open the Stock search grid and search for the pre-licensed image
- validate the "licensed" label exists on the image
- validate the button for the image says "Save"
- click the Save button
- ensure the image is visible in the Media Gallery with the "licensed" label.

Thoughts?

**Comment by drpayyne** [comment]
@filmaj one of the tests in #443 does the ‘saving’ functionality test for a prelicensed image. 

Actually, both. One is for preview saved and other, without.

**Comment by filmaj** [comment]
OK, excellent. In that case I would say we close this issue in favour of #440 and proceed as you have detailed in your PRs.

**Comment by drpayyne** [comment]
@filmaj, please proceed on closing this. Thank you.

**Comment by sivaschenko** [comment]
@filmaj @drpayyne this task can be closed after the corresponding verification will be merged. If you'd like to add this verification to the test introduced in https://github.com/magento/adobe-stock-integration/pull/637 - please add this issue to the list of resolved issues in that PR

**Comment by filmaj** [comment]
Actually, looking at this again, I think we need a dedicated test for:

1. loading the adobe stock grid as-is
2. assuming the test always uses a specific Adobe IMS account with a specific pre-licensed image in the account
3. searching for the pre-licensed image in the search bar via its file id
4. asserting the licensed label is there

**Comment by filmaj** [comment]
@sivaschenko do you know which test / PR  closed this issue? I do not think this task is resolved.

**Comment by filmaj** [comment]
On the plus side I think this is the last outstanding MFTF task to get to 100%  MFTF coverage on Milestone 2 😄

**Comment by drpayyne** [comment]
Ready to work on the mentioned MFTF test too, @filmaj! Just assign me to it if it does get confirmed. Will work on it in the morning!

**Comment by filmaj** [comment]
This MFTF test is still to be written. The scope of this test is ensuring that in the Media Gallery, licensed labels show up for licensed-and-saved images.

**Comment by filmaj** [comment]
I think essentially the same test as `AdminAdobeStockSavedLicensedImageLocateTest.xml` could be copied and an additional assertion added to the end where we check that the licensed label is applied to images in the grid.
